src/app/common/http-facade.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response } from "@angular/http"
+import { Observable } from "rxjs/Observable";
+
+export interface HttpFacade {

```

Comment:
Normal website path: uses angular/http classes to make requests again IIS Admin host
WAC path: uses WAC/Powershell to invoke local_http.ps1 and make local http request on IIS admin host isself